app/views/groups/index.html.erb

```diff
@@ -22,16 +22,16 @@
         }
       );
     });
-    $(function() {
-      $('#assignment-group-use').click(function(e) {
-        modalAssignmentGroupReUse.open();
-        $('#assignment-group-use-close').click(function(e) {
-          modalAssignmentGroupReUse.close();
+      document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Oh, I noticed that this function can be merged into the above (which is also a `DOMContentLoaded` callback)